### ISSUE TITLE
Fix entity ref for summit session (on Event nodes) 

### ISSUE THREAD

**Issue by ForestMars** [NYC-Camp/website#205]
Last year I set up a _nice_ entity reference view that embedded session node types into the summit event the session was being given at. 

Cf. http://2013.nyccamp.org/event/media-summit

Somehow it seems like this functionality got broken on the current site. Can someone please fix it. As you can see from the above link, it makes a huge difference in letting users see at a glance the schedule of events for a given summit.

**Comment by likewhoa** [comment]
admin/structure/views/view/summit_reference/edit/entityreference_1 is the main issue with this and once we remove the filter on content day, we should be good as I already took care of the display fields. The only other tasks needed here is to reference the event nodes with 'session topics' and 'which summit' field list options.

**Comment by ForestMars** [comment]
Does that mean that
http://www.nyccamp.org/session/web-experience-management
should already be showing up on 
http://www.nyccamp.org/event/npo-ngo-summit-2014 ?

Another example: 
http://www.nyccamp.org/session/marketers-case-drupal
should be showing up on
http://www.nyccamp.org/2014/summits/digital-publishing-media

**Comment by likewhoa** [comment]
no, because the field_which_summit is not working because of said filter. Once we remove this filter, which I can now, already existing nodes can select from it then those pages will have the referenced nodes showing under.

**Comment by likewhoa** [comment]
node, field_which_summit is only available on event content types.

**Comment by likewhoa** [comment]
I when ahead and removed this filter so that which_summit field is populated again, do review and let me know if there is anything else needed.

**Comment by willykaram** [comment]
@ForestMars @likewhoa I'm in the midst of fixing this, was working on it a bit earlier in the week and yesterday/today in connection with the scheduling work. Once it's done and themed I'll post it to multi-dev for review and we can hopefully get it pushed to live tomorrow.
